src/course-outline/CourseOutline.test.jsx

```diff
@@ -483,6 +485,92 @@ describe('<CourseOutline />', () => {
     expect(datePicker).toHaveValue('08/10/2025');
   });
 
+  it('check configure subsection when configure subsection query is successful', async () => {
+    const {
+      findAllByTestId,
+      findByText,
+      findAllByPlaceholderText,
+      findAllByRole,
+      findByRole,
+      findByTestId,
+    } = render(<RootWrapper />);
+    const section = courseOutlineIndexMock.courseStructure.childInfo.children[0];
+    const subsection = section.childInfo.children[0];
+    const newReleaseDate = '2025-08-10T10:00:00Z';
+    const newGraderType = 'Homework';
+    const newDue = '2025-09-10T10:00:00Z';
+    const isTimeLimited = true;
+    const defaultTimeLimitMinutes = 210;
+
+    axiosMock
+      .onPost(getCourseItemApiUrl(subsection.id), {
+        publish: 'republish',
+        graderType: newGraderType,
+        metadata: {
+          visible_to_staff_only: true,
+          due: newDue,
+          hide_after_due: false,
+          show_correctness: false,
+          is_practice_exam: false,
+          is_time_limited: isTimeLimited,
+          exam_review_rules: '',
+          is_proctored_enabled: false,
+          default_time_limit_minutes: defaultTimeLimitMinutes,
+          is_onboarding_exam: false,
+          start: newReleaseDate,
+        },
+      })
+      .reply(200, { dummy: 'value' });
+
+    axiosMock
+      .onGet(getXBlockApiUrl(section.id))
+      .reply(200, section);
+
+    const [currentSection] = await findAllByTestId('section-card');
+    const [firstSubsection] = await within(currentSection).findAllByTestId('subsection-card');
+    const subsectionDropdownButton = firstSubsection.querySelector('#subsection-card-header__menu');
+    expect(subsectionDropdownButton).toBeInTheDocument();
+
+    subsection.start = newReleaseDate;
+    subsection.due = newDue;
+    subsection.format = newGraderType;
+    subsection.isTimeLimited = isTimeLimited;
+    subsection.defaultTimeLimitMinutes = defaultTimeLimitMinutes;
+    section.childInfo.children[0] = subsection;
+    axiosMock
+      .onGet(getXBlockApiUrl(section.id))
+      .reply(200, section);
+
+    await executeThunk(configureCourseSubsectionQuery(
+      subsection.id,
+      section.id,
+      true,
+      newReleaseDate,
+      newGraderType,
+      newDue,
+      true,
+      defaultTimeLimitMinutes,
+      false,
+      false,
+    ), store.dispatch);
+    fireEvent.click(subsectionDropdownButton);
+    const configureBtn = await findByText(cardHeaderMessages.menuConfigure.defaultMessage);
```

Comment:
We can make use of `data-testId` field to avoid using general text like `Configure` to get elements. For example:
```suggestion
    const configureBtn = await within(firstSubsection).findByTestId('subsection-card-header__menu-configure-button');
```